components/Chips/README.md

```diff
@@ -1,44 +1,57 @@
 <!--docs:
-title: "Chips"
+title: "Chip"
```

Comment:
Component name should be Chips in .vars.

Reply:
done.